pkg/shared/clients/nats/default_jetstream_client.go

```diff
@@ -63,9 +63,6 @@ func natsJetStreamConnection(ctx context.Context, url string, natsOptions []nats
 				log.Info("Nats: disconnected")
 			}
 		}),
-		nats.ReconnectHandler(func(nnc *nats.Conn) {
```

Comment:
No harm to keep it?

Reply:
BTW - does like 58 need to be changed?

Reply:
I am defining reconnect handler `inside in_cluster_jetstream_client.go`